frontend/public/texts/tutorial_texts.js

```diff
@@ -201,25 +201,23 @@ export default function getTutorialTexts({ hubName, classes, locale }) {
     },
     filter_and_find_tutorial_text: {
       en: `Click on the "Filter" button to filter the projects, for example by location or
-      category. Choose what you want to filter by and click "Apply" to see the results!`,
+      category.`,
       de: `Klicke auf die Schaltfläche "Filter", um die Projekte zu filtern, z. B. nach Ort oder
       Kategorie. Wählen aus, wonach du filtern möchtest und klicke auf "Anwenden", um die Ergebnisse zu sehen!`,
```

Comment:
```suggestion
      Kategorie.`,
```
This is the German counterpart of the text you just removed :)